index.js

```diff
@@ -363,11 +373,11 @@ function replaceRequireAndDefine(code, amdPackages, amdModules) {
         }
 
         // Dealing with ember-auto-import eval
-        if (node.callee.name === 'eval') {
+        if (node.callee.name === 'eval' && node.arguments[0].type === 'Literal' && typeof node.arguments[0].value === 'string') {
```

Comment:
What's the scenario that lead you to add these extra tests?

Reply:
Cases where the eval string is stored in a variable.

In my case, from `sinon-esm.js`
```
function callTimer(clock, timer) {
    ...
    /* eslint no-eval: "off" */
    eval(timer.func);
    ...
}
```

`node.arguments[0].value` was null. So trying to pass value to `replaceRequireAndDefine` ended poorly

Reply:
Thx

Reply:
And changelog added.